### PR TITLE
🔖 Add create new hook docs

### PR DESCRIPTION
**Pull request by MiZiet**
image

### CODE REVIEW

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -59,3 +59,29 @@ Below is a simple example:
 
 Full example code is available `here <https://github.com/EthWorks/useDapp/tree/master/packages/example>`_.
 
+Creating new hook
+------------
```
Comment (JustynaBroniszewska) [inline docs/source/getting-started.rst:63]:
```suggestion
Creating new hook
-------------------
```

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -59,3 +59,29 @@ Below is a simple example:
...
+
+Creating a custom hook with use of our core hooks is very easy, as example let's write a *useEtherBalance* hook:
+
```
Comment (JustynaBroniszewska) [inline docs/source/getting-started.rst:66]:
```suggestion
Creating a custom hook with the use of our core hooks is very easy, as an example let's write a *useEtherBalance* hook:

```

File: docs/source/getting-started.rst [hunk 3]
```diff
@@ -59,3 +59,29 @@ Below is a simple example:
...
+    //encodes data for a function call
+
+    const etherBalance = useChainCall(
+      getEthBalanceCall && multicallAddress &&
+        { address: multicallAddress, data: getEthBalanceCall }
+    ) //makes a chain call
+
+    return { etherBalance: etherBalance !== undefined
+      ? BigNumber.from(etherBalance)
+      : undefined } //returns balance
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:82]: You will need useMemo an explanation why you need useMemo.

File: docs/source/getting-started.rst [hunk 4]
```diff
@@ -59,3 +59,29 @@ Below is a simple example:
...
+------------
+
+Creating a custom hook with use of our core hooks is very easy, as example let's write a *useEtherBalance* hook:
+
+.. code-block:: javascript
+
+  export function useEtherBalance(address: string | Falsy) {
+    const multicallAddress = useMulticallAddress() // gets address of multicall contract
+    const getEthBalanceCall = address &&
+      MultiCallABI.encodeFunctionData('getEthBalance', [address])
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:72]: It is super non-obvious why we use getEthBalance - need explanation.

Comment (marekkirejczyk) [reply docs/source/getting-started.rst:72]: Also a paragraph on what the hell is multicall is a good idea ;)

File: docs/source/getting-started.rst [hunk 5]
```diff
@@ -59,3 +59,29 @@ Below is a simple example:
...
+
+    const etherBalance = useChainCall(
+      getEthBalanceCall && multicallAddress &&
+        { address: multicallAddress, data: getEthBalanceCall }
+    ) //makes a chain call
+
+    return { etherBalance: etherBalance !== undefined
+      ? BigNumber.from(etherBalance)
+      : undefined } //returns balance
+  }
```
Comment (JustynaBroniszewska) [inline docs/source/getting-started.rst:83]:
```suggestion
  export function useEtherBalance(address: string | Falsy) {
    const multicallAddress = useMulticallAddress() // gets address of multicall contract
    const getEthBalanceCall = address &&
      MultiCallABI.encodeFunctionData('getEthBalance', [address])
    // encodes data for a function call

    const etherBalance = useChainCall(
      getEthBalanceCall && multicallAddress &&
        { address: multicallAddress, data: getEthBalanceCall }
    ) // makes a chain call

    return { etherBalance: etherBalance !== undefined
      ? BigNumber.from(etherBalance)
      : undefined } // returns balance
  }
```

File: docs/source/getting-started.rst (hunk 3 above)
Comment (MiZiet) [reply docs/source/getting-started.rst:82]: Should I wrap the return in useMemo? should it be done in the hook or just in docs?

**Comment by JustynaBroniszewska** [comment]
Btw creating custom hooks shouldn't be the first example.

**Comment by MiZiet** [comment]
image

File: docs/source/getting-started.rst (hunk 2 above)
Comment (MiZiet) [reply docs/source/getting-started.rst]: ✅

File: docs/source/getting-started.rst (hunk 3 above)
Comment (MiZiet) [reply docs/source/getting-started.rst]: We moved to useTokenBalance and it's not relevant anymore

File: docs/source/getting-started.rst (hunk 4 above)
Comment (MiZiet) [reply docs/source/getting-started.rst]: We moved to useTokenBalance and it's not relevant anymore

File: docs/source/getting-started.rst (hunk 5 above)
Comment (MiZiet) [reply docs/source/getting-started.rst]: ✅

**Comment by MiZiet** [comment]
image

File: docs/source/guide.rst [hunk 6]
```diff
@@ -23,8 +23,31 @@ If that is too complex consider using a custom backend or The Graph.
...
+    const [tokenBalance] = useContractCall(
+      ERC20Interface, // ABI interface of the called contract
+      tokenAddress, // On-chain address of the deployed contract
+      'balanceOf', // Method to be called
+      address && [address] // Method arguments - address to be checked for balance
+    ) ?? []
+    return tokenBalance
+  }
+
+As you can see by using *useContractCall* getting a token balance is simple.
```
Comment (marekkirejczyk) [inline docs/source/guide.rst:42]: Opinion
```suggestion
```

File: docs/source/guide.rst [hunk 7]
```diff
@@ -23,8 +23,31 @@ If that is too complex consider using a custom backend or The Graph.
 Custom hooks
 ************
 
-Tesing hooks
-************
+Creating a custom hook with the use of our core hooks is very easy, as an example let's write a *useTokenBalance* hook.
```
Comment (marekkirejczyk) [inline docs/source/guide.rst:26]:
```suggestion
Creating a custom hook with the use of our core hooks is straightforward, as an example let's write a *useTokenBalance* hook.
```

File: docs/source/guide.rst (hunk 6 above)
Comment (MiZiet) [reply docs/source/guide.rst]: ✅

File: docs/source/guide.rst (hunk 7 above)
Comment (MiZiet) [reply docs/source/guide.rst]: ✅

**Review by marekkirejczyk — APPROVED** [review]

**Pull request merged**